src/Config/Config.ts

```diff
@@ -22,6 +22,7 @@ export default class Config implements IConfigParams {
 
   constructor(params: IConfigParams) {
     this.autostart = params.autostart || true;
+    this.ms = params.ms || 4000;
```

Comment:
There is another assignment to `this.ms` [here](https://github.com/ethereum-alarm-clock/eac.js-client/blob/6125a069b98562412c99b2aa692708dc4d91877c/src/Config/Config.ts#L70). One of them should be removed imo.

Reply:
Thanks, you're right.